### DISCUSSION TITLE
May I ask where the NACK Report logic is handled?

### DISCUSSION THREAD

**Discussion by lokenetwork** [paullouisageneau/libdatachannel#1060] [Q&A] [answered]
I use media-sender to send the h264 video to my sfu server, server had send the nack rtcp to media-sender process.

I make a breakpoint in RtcpNackResponder::incoming(), why not work, It is not stop at my breakpoint, Why?
![3TH$WFJ_B 6Y3O_JZ)%6KM](https://github.com/paullouisageneau/libdatachannel/assets/5117703/799d911c-d561-48f1-a9aa-8c742929134d)

![)D)N0H%2$ 4 3U9`N7RV4WH](https://github.com/paullouisageneau/libdatachannel/assets/5117703/1b3f5993-bb74-4ee1-8014-3061bca1a4bb)

**Comment by chobie** [comment, marked as answer]
as far as i known, it requries to AddToChain the RtcpNackResponder.

https://github.com/paullouisageneau/libdatachannel/blob/499398a4b7522d91084cb027ee2a7fc4e106519d/examples/streamer/main.cpp#L218

**Reply by lokenetwork** [reply]
Thank you, You save me again.
